pkg/daemon/ceph/agent/flexvolume/server.go

```diff
@@ -271,6 +271,13 @@ func copyFile(src, dest string) error {
 	if err != nil {
 		return errors.Wrapf(err, "error copying file from %s to %s", src, dest)
 	}
+	defer func() error {
```

Comment:
The files already have a defer action to close them, and we don't want to call Sync() twice. We want to run `Close()` after the `destFile.Sync()`, but not in a defer statement. Also we have to close both destFile and srcFile.